docs/src/main/tut/docs/akkahttp.md

```diff
@@ -1,5 +1,112 @@
 ---
```

Comment:
We have adapted the convention to call all our doc files `README.md` and to put them in a directory with a more descriptive name, which should make it easier to read the documentation on github itself.